### PR TITLE
Added 90movies.md

### PR DESCRIPTION
**Pull request by estefaniaramirezmnt**

### CODE REVIEW

File: 90movies.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- Forrest Gump
+- Fight Club
+- The Matrix
+- Fargo
+- Jurassic Park
+- Toy Story
+- Titanic
+- Se7en
+- The Silence of the Lambs
+- American Beauty
```
Comment (juanlu-sanz) [inline 90movies.md:13]: These might be unnecessary. The criteria mentioned only top 3

**Review by juanlu-sanz — CHANGES_REQUESTED** [review]
There is some changes to be done, have a look

File: 90movies.md (hunk 1 above)
Comment (estefaniaramirezmnt) [reply 90movies.md]: Fixed!

**Review by juanlu-sanz — APPROVED** [review]
LGTM!

**Pull request merged**